vignettes/texreg.Rnw

```diff
@@ -504,6 +504,7 @@ Class			& Package			& Description \\
 `\code{netlogit}'		& \pkg{sna}			& Logistic regression for networks \\
 `\code{nlme}'			& \pkg{nlme}			& Nonlinear mixed-effects models \\
 `\code{ols}'			& \pkg{rms}			& Ordinary least squares linear models \\
+`\code{panelAR}'			& \pkg{panelAR}			& AR(1)-autocorrelation and panel heteroskedasticity/contemporaneous correlation\\

```

Comment:
can you please replace the note with: "Linear models on panel data structures in the presence of AR(1)-type autocorrelation" ?